src/appMain/sdl_preloaded_pt.json

```diff
@@ -3475,13 +3479,72 @@
                     "maxsize": 100,
                     "since": "5.0"
                 },
+                {
+                    "name": "climateData",
+                    "params": [
+                        {
+                            "name": "externalTemperature",
+                            "key": "OEM_REF_EXT_TEMP",
+                            "params": [
+                                {
+                                    "name": "unit",
+                                    "key": "OEM_REF_EXT_TEMP_UNIT",
+                                    "type": "TemperatureUnit",
+                                    "mandatory": true
+                                },
+                                {
+                                    "name": "value",
+                                    "key": "OEM_REF_EXT_TEMP_VALUE",
+                                    "type": "Float",
+                                    "mandatory": true
+                                }
+                            ],
+                            "type": "Struct",
+                            "mandatory": false
+                        },
+                        {
+                            "name": "cabinTemperature",
+                            "key": "OEM_REF_CABIN_TEMP",
+                            "params": [
+                                {
+                                    "name": "unit",
+                                    "key": "OEM_REF_EXT_TEMP_UNIT",
+                                    "type": "TemperatureUnit",
+                                    "mandatory": true
+                                },
+                                {
+                                    "name": "value",
+                                    "key": "OEM_REF_EXT_TEMP_VALUE",
```

Comment:
```suggestion
                                    "key": "OEM_REF_EXT_TEMP_VALUE",
```
```suggestion
                                    "key": "OEM_REF_CABIN_TEMP_VALUE",
```

Reply:
Done in 13364d006fbe906bff31e55c50fc65457d881c76